lib/generators/revise_auth/model_generator.rb

```diff
@@ -40,12 +36,12 @@ def migration_path
       end
 
       def model_path
-        @model_path ||= File.join("app", "models", "#{file_path}.rb")
+        @model_path ||= File.join("app", "models", "user.rb")
       end
 
       def model_attributes
         [
-          "email:string:index",
+          "email:string:uniq",
```

Comment:
with this we avoid changing the index in the migration. Unfortunately there's nothing like this for not null columns

Reply:
I wish this PR https://github.com/rails/rails/pull/46900 by @fractaledmind could be considered